magicctapipe/resources/lidar_reports_clouds_ST.03.16.yaml

```diff
@@ -0,0 +1,533 @@
+units:
+  timestamp: 'iso'
```

Comment:
this file is for a very specific dataset, therefore there is no point of posting it into the general repository